src/scout_apm/instruments/elasticsearch.py

```diff
@@ -84,11 +84,14 @@ def instrument_client(self):
 
         for method_str in self.__class__.CLIENT_METHODS:
             try:
-                code_str = """
+                code_str = """\
```

Comment:
This is just to skip an unnecessary blank line in the temporary code, it makes line numbers a bit more understandable.

Reply:
👍